kie-build-tools/pom.xml

```diff
@@ -10,7 +10,9 @@
     <relativePath>../kie-user-bom-parent/pom.xml</relativePath>
   </parent>
 
+  <groupId>org.kie</groupId>
   <artifactId>kie-build-tools</artifactId>
+  <packaging>pom</packaging>
```

Comment:
Is this really correct? The module packages findbugs resources, not sure if using only pom would not exclude them?

Reply:
removed - was wrong!